1_fetch/log/1_fetch_mesonet_tasks.ind

```diff
@@ -0,0 +1,4 @@
+1_fetch/out/mesonet_data_20191220_00.rds: 86d951fff09617d91dc0d40b1ddef325
+1_fetch/out/mesonet_data_20191220_12.rds: deaecbb98d1af2d333d75721e009c2ad
+1_fetch/out/mesonet_data_20191221_00.rds: 8d8460a31640e754085257023f57f494
```

Comment:
We might be able to use a larger timestep than this — looks like you get forecasts from 2-3 days back from the center date

Reply:
Larger as in bigger number (24 hrs) or larger as in more timesteps (4 hrs)

Reply:
Bigger number